package.json

```diff
@@ -147,7 +150,9 @@
     "tests": "ava --verbose \"**/__tests__/*.js\"",
     "pretest": "npm run lint",
     "test": "npm run tests && npm run docs",
-    "test-with-coverage": "nyc npm run tests && npm run docs",
+    "test-with-coverage": "nyc npm run tests && npm run docs && npm run boilerplate",
+    "preboilerplate": "rimraf boilerplate-test",
+    "boilerplate": "mkdirp boilerplate-test && cd boilerplate-test && npm init -f && npm install ../ && ./node_modules/.bin/statinamic setup -t",
```

Comment:
can you also add "npm run boilerplate" in "test" ? Btw, maybe rename boilerplate to "test-boilerplate"